modules/user/user.service.js

```diff
@@ -13,15 +13,15 @@ class UserService {
     return User.find();
   }
 
-  async getUserById(id, auth) {
-    const user = await User.findById(id);
+  async getUserById(auth) {
```

Comment:
I think we should have two methods:
- get the user by id from the decoded token
- get the user by id provided in URL (useful for admin when the owner wants to see some registered user)

Reply:
added